boards/arm/nrf9160_innblue11/doc/index.rst

```diff
@@ -0,0 +1,203 @@
+.. _nrf9160_innblue11:
+
+nRF9160-INNBLUE11
+################

```

Comment:
Title underlines need to be at least as long as the title, so add one more ``#``